src/msw/utils.cpp

```diff
@@ -1328,6 +1328,8 @@ wxOperatingSystemId wxGetOsVersion(int *verMaj, int *verMin)
         *verMaj = s_version.verMaj;
     if ( verMin )
         *verMin = s_version.verMin;
+    if ( verMicro )
+        *verMicro = 0;
```

Comment:
This should probably be the build number in MSW. Especially now with Windows 10 it seems to be actively used by microsoft and it might be intressting to check for it.